HW04/Tatsiana_Zenkovich/spec/my_methods.rb

```diff
@@ -0,0 +1,41 @@
+module MyArray
+  # rubocop:disable Style/For: Prefer each over for
+  def my_each
+    if block_given?
+      for i in (0..length - 1)
+        proc.call(self[i])
+      end
+      return self
+    end
+    return to_enum(:my_each) unless block_given?
```

Comment:
Same thing but a little shorter
```suggestion
      return to_enum(:my_each) unless block_given?

      for i in (0..length - 1)
        proc.call(self[i])
      end
      self

```

Reply:
corrected, as you suggested, thanks!